greendao/src/main/java/com/epam/greendao/GreenDAO.java

```diff
@@ -0,0 +1,106 @@
+package com.epam.greendao;
+
+import android.content.Context;
+
+import com.epam.benchmark.IEntity;
+import com.epam.benchmark.IStorage;
+
+import java.io.IOException;
+import java.util.List;
+
+import de.greenrobot.dao.query.QueryBuilder;
+import de.greenrobot.dao.query.WhereCondition;
+import de.greenrobot.daogenerator.DaoGenerator;
+import de.greenrobot.daogenerator.Entity;
+import de.greenrobot.daogenerator.Schema;
+
+
+/**
+ * Created by shulha_dmytro on 1.10.15.
+ */
+public class GreenDAO implements IStorage {
+
+    public static final String GREENDAO = "greendao";
+    private ModelDao dao;
+
+    public static void main(String[] args) {
+        try {
+            Schema schema = new Schema(1, "com.epam.greendao");
+            schema.enableKeepSectionsByDefault();
+            Entity entity = schema.addEntity("Model");
+
+
+            entity.addStringProperty("id").primaryKey();
+            entity.addIntProperty("index");
+            entity.addBooleanProperty("isActive");
+            entity.addStringProperty("picture");
+            entity.addStringProperty("name");
+            entity.addStringProperty("company");
+            entity.addStringProperty("email");
+            entity.addStringProperty("about");
+            entity.addStringProperty("registered");
+            entity.addDoubleProperty("latitude");
+            entity.addDoubleProperty("longitude");
+
+            new DaoGenerator().generateAll(schema, "greendao/src/main/java/");
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    @Override
+    public void init(Context context) {
+        DaoMaster.DevOpenHelper devOpenHelper = new DaoMaster.DevOpenHelper(context, GREENDAO, null);
+        DaoMaster daoMaster = new DaoMaster(devOpenHelper.getWritableDatabase());
+        DaoSession daoSession = daoMaster.newSession();
+        dao = daoSession.getModelDao();
+    }
+
+    @Override
+    public void save(Context context, List<IEntity> entities) {
+        for (IEntity entity : entities) {
```

Comment:
Please use transaction here to increase performance
http://stackoverflow.com/questions/12748899/greendao-40-seconds-to-insert-600-records